cmd/kubectl-package/validatecmd/validate.go

```diff
@@ -53,6 +53,8 @@ func NewCmd(validator Validator) *cobra.Command {
 			return fmt.Errorf("validating package: %w", err)
 		}
 
+		fmt.Fprintln(cmd.OutOrStdout(), "Package validated successfully!")
```

Comment:
Do you think offloading this into a `const` at the top of the file would make sense?
My thought is: I'ts not a magic string per-say but having a name like `validationSuccessMessage` highlights its semantical meaning. And also highlights that this string will become part of the cli "API".

WDYT?

Reply:
Yup, it makes sense to me as well.

Reply:
![image](https://github.com/package-operator/package-operator/assets/5539202/c97a5ef1-402c-45f9-adaa-8166b35f786e)

Reply:
looks good to me! :) Thanks